docs/projects/fps/tutorial.md

```diff
@@ -240,79 +385,90 @@ namespace Fps
+No, exactly how entities are represented on each of your workers is up to you.
 
-This is because the FPS Starter Project uses the GDK GameObject Creation package as part of the MonoBehaviour workflow. To find out more you can read up about the [MonoBehaviour workflow]({{urlRoot}}/content/intro-workflows-spatialos-entities).<%(/Expandable)%>
+The GDK also offers an [ECS workflow]({{urlRoot}}/content/intro-workflows-spatialos-entities) which represents them as a grouping of Unity ECS entity and components. 
 
-<%(#Expandable title="What's the best way to create a prefab?")%>Prefabs are the Unity Engine approach for creating templates of GameObject hierarchies. [Creating Prefabs (Unity Documentation)](https://docs.unity3d.com/Manual/CreatingPrefabs.html) explains how to create one.
+If you are more familiar with the traditional Unity GameObject style of development then the GDK provides a [MonoBehaviour workflow]({{urlRoot}}/content/intro-workflows-spatialos-entities) for you.
 
-Freshly created prefabs are initially empty, so you can drag other prefabs or GameObjects onto them to add them to the hierarchy.
+You are not limited to these options either, and can configure your worker to create something very custom when it encounters a particular entity type.
 
-It can often be easiest to drag prefabs into a Scene to edit them - just remember to apply your change (in the Unity Inspector panel) and delete them from the Scene when you are done editing! In upcoming versions of Unity Engine you will be able to make use of [prefab mode](https://blogs.unity3d.com/2018/06/20/introducing-new-prefab-workflows/) for this task.<%(/Expandable)%>
+<%(/Expandable)%>
+
+<%(#Expandable title="Can I name the prefab something else?")%>
+
+Your choice of prefab name can be anything, but **must** match the string you used for the entity's `Metadata` component when you wrote the entity template function for this entity.
+
+This is implementation detail from the `AdvancedEntityPipeline` that the FPS Starter Project uses to create GameObjects.
+
+<%(/Expandable)%>
 
 <%(#Expandable title="Wait! Why aren't we removing the callback when the script is disabled?")%>The GDK automatically clears event handlers when a script is disabled, therefore you do not need to manually remove the `OnHealthPickupComponentUpdated` callback.<%(/Expandable)%>
 
-### Test your changes
+### Test the client-side representation
+
+Now we've added some game logic to interact with our `HealthPickup` entity we should test our changes.
+
+> We advise using a test-iterate cycle when developing with the GDK for Unity. You can take advantage of the quick iteration time afforded by running multiple workers in your Unity Editor.
+
+**Step 1.** In your Unity Editor, launch a local deployment of your game by selecting **SpatialOS** > **"Local launch"** or using the shortcut `Ctrl + L`/`Cmd + L`.
```

Comment:
do we need "" around the `Local launch`?

Reply:
Nope